pkg/ebpf/c/tracer-ebpf.c

```diff
@@ -424,8 +424,10 @@ static void update_conn_stats(conn_tuple_t* t, size_t sent_bytes, size_t recv_by
 
     // If already in our map, increment size in-place
     if (val != NULL) {
-        __sync_fetch_and_add(&val->sent_bytes, sent_bytes);
-        __sync_fetch_and_add(&val->recv_bytes, recv_bytes);
+        if (sent_bytes)
```

Comment:
nit: can we make sure to add braces for these if-blocks here, even though it's only a single expression.

Why/how is LLVM registering a constant of zero for this?

Reply:
@sunhay Braces added.

The call path looks like this : kprobe__tcp_sendmsg -> handle_message(&t, size, 0) -> update_conn_stats(t, sent_bytes, recv_bytes)
In this example, the `recv_bytes` in `update_conn_stats` is the 0 constant passed from `kprobe__tcp_sendmsg`